docs/source/commands/init.mdx

```diff
@@ -16,13 +16,64 @@ Running `rover init` starts a short wizard that helps you create a new graph.
 rover init
 ```
 
+<ExpansionPanel title="Experiencing issues?">
+
+    - If you experience a crash, run `rover config clear` and then run `rover config auth` before running `rover init` again.
```

Comment:
Thoughts on linking the issue for more details on what kind of crash we're talking about?

```suggestion
    - If you experience [a crash](https://github.com/apollographql/rover/issues/2521), run `rover config clear` and then run `rover config auth` before running `rover init` again.
```

Reply:
Good question @Meschreiber -- since Matt Hawkins recently released a [fix](https://github.com/apollographql/rover/pull/2533) for this issue this unfurl/expansion is no longer necessary 🎉 This `ExpansionPanel` section can now be omitted.